cmd/nerdctl/container_inspect.go

```diff
@@ -52,74 +47,33 @@ func newContainerInspectCommand() *cobra.Command {
 	return containerInspectCommand
 }
 
+var validModeType = map[string]bool{
+	"native":       true,
+	"dockercompat": true,
+}
```

Comment:
nit: probably we need to find a better place and define these mode as const. I remember I saw them in other places as well.

Reply:
Agreed. Should I mark these out as an issue to be refactored later? The place that I saw these were in `cmd/nerdctl/inspect.go` and I made these to match the style found there.

Reply:
Just noticethere is a copy in `cmd/nerdctl/inspect.go`. Then you can reuse it here instead making a new copy, since they're in the same package.

Reply:
It is a bit different, mine is to verify the mode (native/dockercompat), `inspect.go` is for the type (image/container).

Reply:
I see, then no change requested here :)